pyCGM_Single/pyCGM.py

```diff
@@ -254,31 +270,38 @@ def hipJointCenter(frame,pel_origin,pel_x,pel_y,pel_z,vsk=None):
 
     return hip_JC
 
-def hipAxisCenter(l_hip_jc,r_hip_jc,pelvis_axis):
-    """Calculate the hip joint axis function.
 
-    Takes in a hip joint center of x,y,z positions as well as an index.
-    and takes the hip joint center and pelvis origin/axis from previous functions.
-    Calculates the hip axis and returns hip joint origin and axis.
+def hipAxisCenter(l_hip_jc, r_hip_jc, pelvis_axis):
+    """Calculate the hip joint axis.
+
+    Takes in a hip joint center of x,y,z positions and pelvis axis, and
+    calculates the hip axis and returns the hip joint origin and axis.
+
+    Hip center axis: Computed by taking the mean at each x,y,z axis of the
+    left and right hip joint center.
 
-    Hip center axis: Computed by taking the mean at each x,y,z axis of the left and right hip joint center.
-    Hip axis: Computed by getting the summation of the pelvis and hip center axes.
+    Hip axis: Computed by getting the summation of the pelvis and hip center
```

Comment:
its mean, not sum, but this is already stated above. what is this second description?

Reply:
The hip center is calculated by calculating the mean, but the axis is calculated by adding the pelvis and hip center

Reply:
Does this return the hip center, hip center axis, hip joint center, or pelvis axis.... all of these terms are being used in the same docstring.  In the main description it says hip axis, in the parameters it says pelvis. 

Please make the description consistent.  `Hip center axis` is not the joint center, but its not clear what it is (there is a comment in the code that says what it is).  `hip axis` means the xyz vectors of the middle of the two hip axis, with the origin added back.  It would be good to make this all more clear in phrasing.

Reply:
It returns the hip center axis and hip axis. I just added a new line between `Hip center axis` and `Hip axis` but I'll remove that for consistency. The hip center axis and hip axis are both still in the documentation.